src/solvers/dg_simplices/analysis.jl

```diff
@@ -0,0 +1,75 @@
+# !!! warning "Experimental features"
+
+function calc_error_norms(func, u::StructArray, t, analyzer,
+                          mesh::AbstractMeshData{Dim}, equations, initial_condition,
+                          dg::DG{<:RefElemData{Dim}}, cache, cache_analysis) where {Dim}
```

Comment:
If you rename the solver type in the `dg.jl` file, you could use
```suggestion
                          dg::VertexMappedDG, cache, cache_analysis) where {Dim}
```
or something similar here

Reply:
Using `MultiDG{...}`